docs/usage/from-a-terminal.rst

```diff
@@ -199,37 +281,34 @@ from each given input files.
     information and participate in it's development.
 
 
+------
+
 :code:`--wildcard`
 """"""""""""""""""
 .. versionadded:: 3.3.0
 
-Activates or disables the decoding of wildcards for each given input files.
-
-**Default value:** :code:`wildcard: False`
+The flag to use when your source(:code:`-f`) of domains starts with a wildcard.
 
-.. code-block:: bash
+This flag will subtract the :code:`*.$DOMAIN` and test the :code:`$DOMAIN`
+according to the test arguments given.
 
-    '*.wildcard.me'
-    'wildcard.me'
+**Default value:** :code:`wildcard: False`
 
-These are examples of when to use this argument. The first one will
+A examples of when to use this argument. The first one will
```

Comment:
`A examples` may not be correct. `A` and `examples` (plural are incompatible)

Reply:
Is it ok if I add the missing `s` ??

```rst
As examples of when to use this argument. The first one will
```

Reply:
Yes it's okay.